ratpack/build.gradle

```diff
@@ -14,6 +14,8 @@ if (!JavaVersion.current().java8Compatible) {
 
 apply plugin: "io.ratpack.ratpack-java"
 apply plugin: 'java'
+apply plugin: 'groovy'
+apply plugin: 'io.ratpack.ratpack-groovy'

```

Comment:
Why do we need both the gradle file AND the pom?

Reply:
gradle was already there in the Ratpack module to let reader build and run it as gradle project